### ISSUE TITLE
Issue getting up and running

### ISSUE THREAD

**Issue by sevenseacat** [keathley/wallaby#25]
Pretty sure I've followed everything in the README and got up to actually writing some tests; but running into issues. I'm seeing this error:

```
Erlang/OTP 18 [erts-7.2.1] [source] [64-bit] [smp:4:4] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]

test/acceptance/create_projects_test.exs:6: warning: variable session is unused

Forum.CreateProjectTest
  * alternate formatRequest to pry #PID<0.304.0> at test/acceptance/create_projects_test.exs:20

        visit session, "/"

        IEx.pry
        click_link session, "New Project"


Allow? [Yn] n
23:51:33.726 [error] Postgrex.Protocol (#PID<0.258.0>) disconnected: ** (DBConnection.Error) ownership timeout
23:51:34.456 [error] Postgrex.Protocol (#PID<0.257.0>) disconnected: ** (DBConnection.Error) ownership timeout
```

And then after 60 second the test times out and fails. The first `visit` is fine, but inspecting `text(session)` in the IEx.pry returns an empty string. Something is missing either in my configuration, or in the setup steps, and I'm not sure what it is :)

I've pushed my test app to GitHub here: https://github.com/sevenseacat/test_wallaby

**Comment by keathley** [comment]
Thanks for pushing up the test app. I'll take a look at it.

**Comment by keathley** [comment]
so `text/1` takes a `Node` and not a `Session` which is why you get an empty string. The actual errors that you're getting are from your `ConnCase` test case: `Ecto.Adapters.SQL.restart_test_transaction(Forum.Repo, [])` 

`restart_test_transaction` isn't supported in ecto beta 2.

**Comment by keathley** [comment]
I'm going to close this for now since it seems like its an unrelated problem. If you still have issues feel free to re-open.

**Comment by sevenseacat** [comment]
I might have to re-open it because my acceptance tests aren't using that ConnCase - my AcceptanceCase helper has the correct line:

`:ok = Ecto.Adapters.SQL.Sandbox.checkout(Forum.Repo)`

And the error persists (even after I update the ConnCase).

I'll take another look at it this evening, I may have out-of-date something else in there causing problems.

**Comment by sevenseacat** [comment]
Would you be able to push up a sample app using the gem and having a test running? I've tried recreating a Phoenix app using the latest version of everything and no matter what I do, I can't shake this `[error] Postgrex.Protocol (#PID<0.294.0>) disconnected: ** (DBConnection.Error) ownership timeout` error.

**Comment by keathley** [comment]
I've dug into this a bit more. It is a Wallaby issue. The problem is that if there isn't a max_wait_time set then Wallaby never times out looking for the elements on the page. Eventually that manifests as a ownership error because we've held ownership for too long.

The quick fix is to add: `Application.put_env(:wallaby, :max_wait_time, 3_000)` to your `test_helper.exs`. That will cause you to see the correct error. You will still have issues with your test as I don't think you're actually going to the correct page. But it should at least help you get moving. I'll fix this and push out a new version as soon as I can.

Thanks for being persistent about this issue. :heart:

**Comment by keathley** [comment]
This has been resolved in #32
